tasks/main.yml

```diff
@@ -14,5 +14,8 @@
     owner: root
     group: root
     mode: 0750
-  notify:
-    - Execute lint and copy script
+
+- name: Execute lint and copy script
```

Comment:
Sorry for being pedantic but the task name is misleading (should be `Execute lint script`).